hybrid-tables/app/README.md

```diff
@@ -0,0 +1,20 @@
+## Welcome to your First Snowflake Native App!
```

Comment:
We should change this readme, it can be short but it shouldn't be the default that comes with the template